src/blocks/social/components/edit.js

```diff
@@ -85,11 +88,12 @@ class Edit extends Component {
 			borderRadius: borderRadius ? borderRadius + 'px' : undefined,
 			backgroundColor: ! hasColors && ! isMaskStyle ? backgroundColor.color : undefined,
 			color: ! hasColors ? this.getTextColor( isMaskStyle ) : undefined,
+			padding: isCircularStyle && padding ? padding + 'px' : undefined,
```

Comment:
We can actually drop the ternary operator here and just use the `&&` operator for the same effect. The difference is we're returning `false` instead of `undefined` if `isCircularStyle` is false.

```suggestion
			padding: isCircularStyle && padding + 'px',
```